src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/UpToDate/BuildUpToDateCheck.cs

```diff
@@ -27,6 +27,9 @@ internal sealed partial class BuildUpToDateCheck
         internal const string FastUpToDateCheckIgnoresKindsGlobalPropertyName = "FastUpToDateCheckIgnoresKinds";
         internal const string TargetFrameworkGlobalPropertyName = "TargetFramework";
 
+        // This analyzer fires for comparisons against the following constants. Disable it in this file.
+        #pragma warning disable CA1820 // Test for empty strings using string length
```

Comment:
This suppression is unrelated to this PR but cleans up some diagnostics in the IDE.